### ISSUE TITLE
class "default" rule for syntax highlighting

### ISSUE THREAD

**Issue by 3DEsprit** [decosoftware/deco-ide#100]
Anytime you use the "default" keyword with a class, the syntax highlighting is laid out as if there is an error. The classname is shown in 'white' instead of orange. In addition it also effects all of the methods inside the class as well. If you're someone like me who likes to put them inline with the class call as much as possible, this can throw you off. 

Without **default**: http://d.pr/i/1aWgw
With **default**: http://d.pr/i/1fMTQ

It seems that with a separate line for a default export, the use of default doesn't make a difference (Object in white always), and makes me wonder if there is an inconsistency as well. Anyways, it's not a huge deal, but wanted to at least point it out for the record.

My apologies if this is something fixed in the next release. On the same note, is there a place to check what fixes are in the new release so to not overlap issues?

**Comment by dabbott** [comment]
Good catch. I will fix this. The CodeMirror parser is missing some of this ES6/ES7 stuff.

We tentatively started using ZenHub (a browser plugin that tweaks the github site) to track releases, but I haven't been that diligent about managing things yet. For now, I'd say don't worry about overlapping issues - there have only been one or two duplicates, and I can manage merging them. You can always check closed pull requests to see what will go in the next release, if you're curious.

**Comment by 3DEsprit** [comment]
Ah! Ok, I won't worry too much, but I may find myself looking at pulls at some point. :wink: :
